tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/Logger.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.Identity.Client;
+
+namespace Microsoft.Identity.Web.Perf.Client
+{
+    public class Logger
```

Comment:
do we need this? are we going to be analyzing the logs on failures, or do we have enough info with the pre-existing dashboards in app services? could be one more thing for someone to have to go through. just a thought.

Reply:
Persisting test runner exceptions is needed I think, although it should be updated to push that data to App Insights rather than a file. The MSAL logging is set to Info level right now, which does generate a lot of logs. I would set it to Error level and also try to update it to push to App Insights. I don't know if we need Info level logging for MSAL in a client, as we do have other related counters.